consensus/safety-rules/src/tests/suite.rs

```diff
@@ -16,114 +17,111 @@ fn make_proposal_with_qc_and_proof(
     round: Round,
     proof: Proof,
     qc: QuorumCert,
-    validator_signer: &ValidatorSigner,
+    signer: &ValidatorSigner,
 ) -> VoteProposal<Round> {
-    test_utils::make_proposal_with_qc_and_proof(round, round, proof, qc, validator_signer)
+    test_utils::make_proposal_with_qc_and_proof(round, round, proof, qc, signer)
 }
 
 fn make_proposal_with_parent(
     round: Round,
     parent: &VoteProposal<Round>,
     committed: Option<&VoteProposal<Round>>,
-    validator_signer: &ValidatorSigner,
+    signer: &ValidatorSigner,
 ) -> VoteProposal<Round> {
-    test_utils::make_proposal_with_parent(round, round, parent, committed, validator_signer)
+    test_utils::make_proposal_with_parent(round, round, parent, committed, signer)
 }
 
-#[test]
-fn test_initial_state() {
+type RoundCallback = fn() -> (Box<dyn TSafetyRules<Round>>, Arc<ValidatorSigner>);
```

Comment:
the naming is so confusing, I'm looking at this for a while and realize round refers to the generic parameter

Reply:
Could call it SafetyRulesRoundCallback or SRRoundCallback